### PR TITLE
Remove mutations from pathology and add spread stat instead and stuff

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
This removes all the stuff related to pathogen mutations.
This includes all the mutations as well as the three stats for them, Mutativeness, Mutation Speed, and Maliciousness.

It also introduces a new stat, Spread, which affects how much spreading symptoms will spread the pathogen.
At 0 Spread, symptoms should not spread at all. At 50 spread, they will spread about twice as much as before.
This affects symptoms differently, for instance, airborne spread symptoms will now have up to twice the rate to trigger.

To balance this, there is now a limited amount of points to spread among the (now three) stats.
This depends on the microbody type, for instance, virii have 100 points to spread right now.
All the stats have been adjusted and capped at 0 and 50 for this.
## Why's this needed? 
I feel like patho mutations never really added much to the game.
As a non-traitor pathologist, you could never rely on them, because they could introduce negative symptoms.
As a traitor pathologist, at their best they were subpar and boring way to get symptoms.
At their worst, they were an overpowered and boring way to get symptoms.
Obtaining a high enough maliciousness to actually get worthwhile symptoms was just silly as well.
I feel like this much randomness if fundamentally bad to have in pathogens, so I have elected to just remove them.

By capping stats between 0 and 50 and giving a limited amount of them their values can actually be meaningful decisions for the pathologist and not just depend on how long they are willing to click + and - buttons for.

The introduction of the Spread stat also allows one to make a non-contagious version of any pathology symptom, which is something I have wished for many times as a non-traitor pathologist.
Now brave crewmembers can use potentially harmful and fun pathogens on themselves without running the risk of affecting other crewmembers that may not consent to such pathogens.

### CODE REVIEW

File: browserassets/js/pathoui-script.js [hunk 1]
```diff
@@ -111,20 +111,14 @@
...
+				$("#manipHelp").html("Advance Speed determines how quickly your pathogen will advance through its stages.");
 				break;
 			case "helpSupThreshold":
-				$("#manipHelp").html("Suppression Threshold determines how hard your pathogen is to suppress. Higher values will require more of the suppressant chem or higher external suppression factors. This is capped at 50 for most purposes. Suggested range: 0-50");
+				$("#manipHelp").html("Suppression Threshold determines how hard your pathogen is to suppress. Higher values will require more of the suppressant chem or higher external suppression factors.");
+				break;
+			case "helpSpread":
+				$("#manipHelp").html("Spread determines how easily your pathogen will spread. How this expresses itself depends on the symptoms, for instance, a symptom that spreads via pathogen clouds might make larger clouds, while a symptom that spreads on hugs might have a higher chance per hug.");
```
Comment (zjdtmkhzt) [inline browserassets/js/pathoui-script.js:121]:
```suggestion
				$("#manipHelp").html("Spread determines how easily your pathogen will spread. How this expresses itself depends on the symptoms, for instance, a symptom that spreads via pathogen clouds might make clouds more often, while a symptom that spreads on hugs might have a higher chance per hug.");
```

File: code/modules/medical/pathology/pathogen_microbodies.dm [hunk 2]
```diff
@@ -74,6 +64,9 @@ datum/microbody
 	// The amount of sequences worth of symptoms the microbody can support. -1 is unlimited
 	var/seqMax = -1
 
+	// The maximum amount of points that can be spread over the various stats
```
Comment (ZeWaka) [inline code/modules/medical/pathology/pathogen_microbodies.dm:67]:
```suggestion
	/// The maximum amount of points that can be spread over the various stats
```

File: code/modules/medical/pathology/pathogen_microbodies.dm [hunk 3]
```diff
@@ -74,6 +64,9 @@ datum/microbody
 	// The amount of sequences worth of symptoms the microbody can support. -1 is unlimited
```
Comment (ZeWaka) [inline code/modules/medical/pathology/pathogen_microbodies.dm:64]:
```suggestion
	/// The amount of sequences worth of symptoms the microbody can support. -1 is unlimited
```

File: code/modules/medical/pathology/pathogen_symptoms.dm [hunk 4]
```diff
@@ -102,21 +102,29 @@ datum/pathogeneffects
...
 	// OVERRIDE: Overriding this is situational. ..() is expected to be called.
 	proc/ondisarm(var/mob/M as mob, var/mob/V as mob, isPushDown, var/datum/pathogen/origin)
-		if (infect_type == INFECT_TOUCH)
+		if (infect_type == INFECT_TOUCH && prob(origin.spread*2))
 			infect_direct(V, origin, "disarm")
 		return 1
 
+	// ongrab(mob, mob, datum/pathogen) : void
+	// TODO: Make this a veto event.
+	// OVERRIDE: Overriding this is situational. ..() is expected to be called.
```
Comment (ZeWaka) [inline code/modules/medical/pathology/pathogen_symptoms.dm:111]: god this comment syntax is curse-d

File: code/modules/medical/pathology/pathogen_symptoms.dm [hunk 5]
```diff
@@ -1894,7 +1892,7 @@ datum/pathogeneffects/malevolent/beesneeze
 		var/obj/item/reagent_containers/food/snacks/ingredient/egg/bee/toThrow = new /obj/item/reagent_containers/food/snacks/ingredient/egg/bee(T)
 		M.visible_message("<span class='alert'>[M] sneezes out a space bee egg!</span> [chosen_phrase]", "<span class='alert'>You sneeze out a bee egg!</span> [chosen_phrase]", "<span class='alert'>You hear someone sneezing.</span>")
 		toThrow.throw_at(target, 6, 1)
-		src.infect_cloud(M, origin, 10)
+		src.infect_cloud(M, origin, origin.spread) // at some point I want the bees to spread this instead
```
Comment (ZeWaka) [inline code/modules/medical/pathology/pathogen_symptoms.dm:1895]:
```suggestion
		src.infect_cloud(M, origin, origin.spread) // TODO: at some point I want the bees to spread this instead
```

**Review by ZeWaka — APPROVED** [review]
a shame mutations didn't lead to good gameplay, easy to revert this if necessary

no changelog imo
approve conditional on review items blah blah

**Pull request merged**